tests/endtoend/test_worker_process_count_functions.py

```diff
@@ -38,7 +38,7 @@ def get_script_dir(cls):
     def get_environment_variables(cls):
         return cls.env_variables
 
-    @testutils.retryable_test(4, 5)
```

Comment:
Why 5 retries? I feel if it doesn't pass in 3 retries then most likely it won't with 5